packages/synapse-interface/slices/bridge/actions.ts

```
@@ -0,0 +1,220 @@
+import { createAsyncThunk } from '@reduxjs/toolkit'
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)